src/NServiceBus.MessagingBridge/Configuration/BridgeConfiguration.cs

```diff
@@ -49,15 +48,6 @@ internal FinalizedBridgeConfiguration FinalizeConfiguration(ILogger<BridgeConfig
                 throw new InvalidOperationException("At least two transports needs to be configured");
             }
 
-            var tranportsWithNoEndpoints = transportConfigurations.Where(tc => !tc.Endpoints.Any())
```

Comment:
Just to confirm:

1. We allow an empty transport so that we will create reverse mappings for it in order to do address translation even though we would not be dispatching any messages to it

2. We are creating all proxy queues on that transport so that older messages in flight can be handled properly, eg. old failures being retried from SC that ends up there

Reply:
>We allow an empty transport so that we will create reverse mappings for it in order to do address translation even though we would not be dispatching any messages to it

The reason we want to allow an empty transport is to support one-way shoveling.

> We are creating all proxy queues on that transport so that older messages in flight can be handled properly, eg. old failures being retried from SC that ends up there

We create the proxy queues to make sure we can shovel messages to other transports that have a defined endpoint. So as long as a user, has an endpoint defined on some other transport, old messages (e.g. retired from SC) will be picked up from these queues by the bridge.